### PR TITLE
dev-libs/freexl: Security bump to 1.0.5

### PR DESCRIPTION
**Pull request by ajakk**
Bug: https://bugs.gentoo.org/648700
Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: John Helmert III <jchelmert3@posteo.net>

Changes:
* EAPI bumped
* Removed incompatible ltprune eclass and replaced the function it provides as documented here: https://devmanual.gentoo.org/eclass-reference/ltprune.eclass/index.html

Tests pass for me.

### CODE REVIEW

File: dev-libs/freexl/freexl-1.0.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
```
Comment (mgorny) [inline dev-libs/freexl/freexl-1.0.5.ebuild:6]: Unused.

File: dev-libs/freexl/freexl-1.0.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+DESCRIPTION="Simple XLS data extraction library"
+HOMEPAGE="https://www.gaia-gis.it/fossil/freexl/index"
+SRC_URI="http://www.gaia-gis.it/gaia-sins/${PN}-sources/${P}.tar.gz"
+
+LICENSE="MPL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~x86"
+IUSE="static-libs"
```
Comment (mgorny) [inline dev-libs/freexl/freexl-1.0.5.ebuild:15]: Please disable static libs unconditionally, they are not needed by any packages and are a security nightmare.

File: dev-libs/freexl/freexl-1.0.5.ebuild (hunk 1 above)
Comment (ajakk) [reply dev-libs/freexl/freexl-1.0.5.ebuild]: Removed

File: dev-libs/freexl/freexl-1.0.5.ebuild (hunk 2 above)
Comment (ajakk) [reply dev-libs/freexl/freexl-1.0.5.ebuild]: Done. Note configuring with `--disable-static` still produces an `.la` file so the deletion in `src_install` is still necessary.

**Comment by ajakk** [comment]
Sorry, I am not familiar with this software at all, and I don't use it, just bumping it due to security concerns. If security issues come up again in the future I will be happy to bump again, of course.

**Comment by mgorny** [comment]
Ok, thanks!

**Pull request closed without merging**